Metricbeat: Apache 2.4.23 field not found errors

I'm getting:

metricbeat[6664]: schema.go:36: Error on field 'total': Key ConnsTotal not found
metricbeat[6664]: schema.go:36: Error on field 'closing': Key ConnsAsyncClosing not found
metricbeat[6664]: schema.go:36: Error on field 'writing': Key ConnsAsyncWriting not found
metricbeat[6664]: schema.go:36: Error on field 'keep_alive': Key ConnsAsyncKeepAlive not found

Running Metricbeat 5.0.1 with Apache 2.4.23 (from https://ius.io/) on Centos 7:

httpd -version
Server version: Apache/2.4.23 (CentOS)
Server built: Jul 22 2016 09:11:01

metricbeat -version
metricbeat version 5.0.1 (amd64), libbeat 5.0.1

curl http://127.0.0.1:81/server-status?auto
127.0.0.1
ServerVersion: Apache/2.4.23 (CentOS) OpenSSL/1.0.1e-fips
ServerMPM: prefork
Server Built: Jul 22 2016 09:11:01
CurrentTime: Wednesday, 30-Nov-2016 05:32:43 NZDT
RestartTime: Tuesday, 29-Nov-2016 20:41:30 NZDT
ParentServerConfigGeneration: 1
ParentServerMPMGeneration: 0
ServerUptimeSeconds: 31873
ServerUptime: 8 hours 51 minutes 13 seconds
Load1: 0.00
Load5: 0.01
Load15: 0.05
Total Accesses: 2181
Total kBytes: 53079
CPUUser: 8.4
CPUSystem: 1.22
CPUChildrenUser: 0
CPUChildrenSystem: 0
CPULoad: .0301823
Uptime: 31873
ReqPerSec: .0684278
BytesPerSec: 1705.3
BytesPerReq: 24921.1
BusyWorkers: 1
IdleWorkers: 9
Scoreboard: __.W._____....................................................................................................................................................................................................................................................
TLSSessionCacheStatus
CacheType: SHMCB
CacheSharedMemory: 512000
CacheCurrentEntries: 0
CacheSubcaches: 32
CacheIndexesPerSubcaches: 88
CacheIndexUsage: 0%
CacheUsage: 0%
CacheStoreCount: 35
CacheReplaceCount: 0
CacheExpireCount: 35
CacheDiscardCount: 0
CacheRetrieveHitCount: 0
CacheRetrieveMissCount: 89
CacheRemoveHitCount: 0
CacheRemoveMissCount: 0

First reported here:
https://discuss.elastic.co/t/metricbeat-apache-2-4-23-field-not-found-errors/67518

--- a/metricbeat/module/apache/status/data.go
+++ b/metricbeat/module/apache/status/data.go
@@ -38,17 +38,17 @@ var (
 			"children_system": c.Float("CPUChildrenSystem"),
 		},
 		"connections": s.Object{
-			"total": c.Int("ConnsTotal"),
+			"total": c.Int("ConnsTotal", s.Optional),
 			"async": s.Object{
-				"writing":    c.Int("ConnsAsyncWriting"),
-				"keep_alive": c.Int("ConnsAsyncKeepAlive"),
-				"closing":    c.Int("ConnsAsyncClosing"),
+				"writing":    c.Int("ConnsAsyncWriting", s.Optional),
+				"keep_alive": c.Int("ConnsAsyncKeepAlive", s.Optional),
+				"closing":    c.Int("ConnsAsyncClosing", s.Optional),
 			},
 		},
 		"load": s.Object{
-			"1":  c.Float("Load1"),
-			"5":  c.Float("Load5"),
-			"15": c.Float("Load15"),
+			"1":  c.Float("Load1", s.Optional),
+			"5":  c.Float("Load5", s.Optional),
+			"15": c.Float("Load15", s.Optional),
 		},
 	}
 )

